src/topojson/topojson.ml

```diff
@@ -26,6 +26,8 @@ module Make (J : Intf.Json) = struct
       | `Null -> [ ("properties", J.null) ]
       | `Obj v -> [ ("properties", J.obj v) ]
 
+    type id = [ `None | `Null | `Obj of (string * json) list ]
```

Comment:
Sorry, I meant the properties should be taken as _inspiration_ not copied ^^" -- I don't think we need to create a new polymorphic variant for this. Using `json option` should be sufficient.